include/bringauto/fleet_protocol/http_client/FleetApiClient.hpp

```diff
@@ -79,8 +81,9 @@ class FleetApiClient {
 	 * @brief Calls the POST function on /status/{company_name}/{car_name} of Fleet v2 HTTP API.
 	 * setDeviceIdentification needs to be used beforehand to set DeviceId, otherwise placeholder values will be used.
 	 * @param statusJson payload data of a status represented by a json as a string
+	 * @param isError optional, if true, the status will be marked as a STATUS_ERROR message
 	 */
-	void sendStatus(const std::string &statusJson);
+	void sendStatus(const std::string &statusJson, std::optional<bool> isError = std::nullopt);
```

Comment:
Do not use std::optional. Define enum class and refer to status type by enumm class.

Similar to

```
enum class StatusType { STATUS, ERROR_STATUS }
void sendStatus(const std::string &statusJson, StatusType statusType = StatusType::STATUS);
```

Because then the func. is invoked as

```
sendStatus(statusJson);
sendStatus(statusJson, StatusType::STATUS_ERROR)
```

Reply:
changed the parameter to a new enum